rules/windows/defense_evasion_posh_encryption.toml

```
@@ -18,6 +18,36 @@ index = ["winlogbeat-*", "logs-windows.*"]
 language = "kuery"
 license = "Elastic License v2"
 name = "PowerShell Script with Encryption/Decryption Capabilities"
+note = """## Triage and analysis
+
+### Investigating PowerShell Script with Encryption/Decryption Capabilities
+
+PowerShell is one of the main tools system administrators use for automation, report routines, and other tasks, making it available for use in various environments, creating an attractive way for attackers to execute code.
+
+PowerShell offers encryption and decryption functionalities that can be abused by attackers for a variety of purposes, from concealing payloads and C2 communications to bypass security protections to ransomware capabilities.
```

The "from ... to" format gets a little confusing with a long phrase, maybe try something like:
```suggestion
PowerShell offers encryption and decryption functionalities that can be abused by attackers for a variety of purposes, such as concealing payloads and C2 communications to bypass security protections and performing ransomware capabilities.
```